README.md

```diff
@@ -7,7 +7,14 @@ Start building your own Svelte application using Nuclia’s search capabilities
 First you need to clone this repository and install dependencies:
 
 ```bash
+# dependant on the method you prefer to do:
 git clone git@github.com:nuclia/ui-starter.git
+# or
+git clone https://github.com/nuclia/ui-starter.git
+# or
+download and uncompress zip directly from the repo
```

Comment:
this is not a command you can run on a terminal, so you should not put it on a code block (or you can keep it here but as a comment, so starting with a `#`).
I think it would probably be better to split this section in 2:
1. "First you need to clone this repository" with the explanations on how to do it
2. "Then you install the dependencies" with the corresponding commands